server/routes/gcp/billing.py

```
@@ -18,49 +19,51 @@ def billing(user_id):
+                try:
+                    if is_bigquery_enabled(project_id, credentials):
+                        logging.info(f"Storing billing data for project: {project_id}")
+                        store_bigquery_data(credentials, project_id, user_id)
+                        processed_projects.append(project_id)
+                    else:
+                        logging.info(f"BigQuery not enabled for project: {project_id}, skipping")
+                except ValueError as e:
+                    if "Permission Denied" in str(e):
+                        logging.warning(f"Permission denied for project {project_id}, skipping.")
+                        continue
+                    raise e
```

Ruff (0.15.14)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)